model/status_board/v1/services_resource.model

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages the collection of services.
+resource Services {
+  // Retrieves the list of services.
+  method List {
+    in out Page Integer = 1
+    in out Size Integer = 100
+    in OrderBy String
+    in Fullname String
+    in Mine Boolean
```

Comment:
What is the meaning of `mine`? I guess it is to filter the results so the user only gets the services that she owns. If that is correct then consider using `search` for that.